puppet/zulip/files/cron.d/delete_deactivated_realms

```diff
@@ -0,0 +1,6 @@
+SHELL=/bin/bash
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+USER=zulip
+
+# Cron job to clear data of deactivated realms.
+0 0 * * * zulip /home/zulip/deployments/current/manage.py delete_deactivated_realms
```

Comment:
It's worth noting that this means that we may take up to 24 hours longer than expected to deleted a realm -- that is, if it expired at 00:00:01, then this cron job will next run nearly 24 hours later to delete it.

That's _probably_ fine -- increasing this frequency would reduce that slop, but 24h-less-immediate deletion seems fine.